### DISCUSSION TITLE
Little Red Arrow: Python

### DISCUSSION THREAD

**Discussion by RunLikeAnAntelope** [LazyVim/LazyVim#4878] [Q&A] [answered]
I am using the default python extra configuration for lazy vim, and I get a little red arrow next to some of the line numbers. Does anyone know what this is or how I could find out what it is? Neovim newb here.

![image](https://github.com/user-attachments/assets/69de54a1-5ff6-4694-ac24-fd769c5dfb4f)

**Comment by folke** [comment, marked as answer]
Git sign for lines you deleted
